dubbo-common/src/main/java/org/apache/dubbo/rpc/model/ApplicationModel.java

```diff
@@ -119,7 +119,7 @@ public static String tryGetApplication() {
             return application;
         }
         Optional<ApplicationConfig> appCfgOptional = getConfigManager().getApplication();
-        return appCfgOptional.isPresent() ? appCfgOptional.get().getName() : null;
+        return appCfgOptional.map(ApplicationConfig::getName).orElse(null);
```

Comment:
Could be converted to a one liner. This will make it more readable and you do not really continue working with `appCfgOptional`.

My advice replace it with 

```
return getConfigManager().getApplication()
.map(ApplicationConfig::getName)
.orElse(null);
```

Reply:
done